Add Migration Guides for Media Gallery (#4873)
GetStream/stream-chat-android: docusaurus/docs/Android/02-migration-guides/02-ui-components/02-media-gallery.mdx

# Media Gallery

> Previously, our XML only opened a gallery with attachments of type image. Now, we can open a gallery with image and/or type of attachment.

Reviewer: ```suggestion
Previously, our XML only opened a gallery with attachments of type image. Now, we can open a gallery with images and/or other types of attachments..
```

> The style of the gallery is the same as the previous one but its name is different.

Reviewer: ```suggestion
The style of the gallery remains the same as before, but its name has changed.
```
